internal/component/otelcol/exporter/loadbalancing/loadbalancing.go

```diff
@@ -135,6 +135,9 @@ type ResolverSettings struct {
 	Static     *StaticResolver     `alloy:"static,block,optional"`
 	DNS        *DNSResolver        `alloy:"dns,block,optional"`
 	Kubernetes *KubernetesResolver `alloy:"kubernetes,block,optional"`
+	// TODO: add AWSCloudMap
+	// https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/27588
+	//AWSCloudMap *AWSCloudMapResolver `alloy:"aws_cloud_map,block,optional"`
```

Comment:
Can we make sure to create an issue for this after this is merged so we can track it officially?

Reply:
I raised a PR for it:
https://github.com/grafana/alloy/pull/755